### ISSUE TITLE
Sunwell Plateau DC (BB #470)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#470]
This issue was migrated from bitbucket.
**Original Reporter:** pinkponeyz
**Original Date:** 05.08.2013 02:17:23 GMT+0000
**Original Priority:** critical
**Original Type:** bug
**Original State:** invalid
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470

<hr>

Solved. Found a fix. Sorry.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** pinkponeyz
**Original Date:** 05.08.2013 06:36:49 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470/#comment-5452075

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** pinkponeyz
**Original Date:** 05.08.2013 07:08:37 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470/#comment-5452334

<hr>

Although, the KJ npcs still aren't spawning and aren't even there when I enter his area.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** PadreWoW
**Original Date:** 05.08.2013 14:25:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470/#comment-5458521

<hr>

Fixed long time ago - use search

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** release_ok
**Original Date:** 04.01.2014 09:20:55 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470/#comment-7584816

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Yooki
**Original Date:** 04.01.2014 13:41:06 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/470/#comment-7586375

<hr>

It would be very kind of the author of this report as well as anyone else who has found or knows of a fix, to publish it in a form of a past in a post if nothing.

Some people should remember this is a community project, at least it was a long time ago, and we should share any information we have that might improve the project in any way.

In short: Could someone who knows what this was about, or if anything is still not as it should be concerning this issue, please let us know?
